server/routes/login.js

```diff
@@ -1,26 +1,30 @@
 var express = require("express");
 var router = express.Router();
 
-var {mongoose, db, Teacher} = require('../database');
+var {mongoose, db, User} = require('../database');
 
-router.get("/", function(req, res, next) {
-    console.log("successfully logged in")
-    res.send("success")
+router.get("/:id", function(req, res, next) {
```

Comment:
this would make more sense in the user.js file. probably all of this would

Reply:
That was my thought as well, the user file is more or less a duplicate of the login file which should be deleted

Reply:
The login.js file has been deleted